### DISCUSSION TITLE
is cppcheck-args option working with 6.21.0

### DISCUSSION THREAD

**Discussion by phani35** [Ericsson/codechecker#3984] [Q&A]
Does "--cppcheck-args" option work with 6.21.0 ?
Do we have any sample cppcheck args file to handle below option.

`--cppcheck-args CPPCHECK_ARGS_CFG_FILE
                        Configuration file to pass cppcheck command line
                        arguments.`

**Comment by bruntib** [comment]
No, unfortunately this flag was missing from CodeChecker. The next release will contain this flag, the PR is here: https://github.com/Ericsson/codechecker/pull/3978
